src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo.cs

```diff
@@ -206,6 +206,9 @@ public SymbolTreeInfo WithChecksum(Checksum checksum)
                 if (ignoreCase || StringComparer.Ordinal.Equals(name, node.Name))
                 {
                     assemblySymbol ??= await lazyAssembly.GetValueAsync(cancellationToken).ConfigureAwait(false);
+                    if (assemblySymbol is null)
```

Comment:
❔ Where's the source of this null value?

Reply:
we are delaying getting the IAssemblySymbol for a particular PE reference, but the compiler does'nt guarantee all pe-references will produce an IAssemblySymbol.